arrow/locales.py

```diff
@@ -78,6 +79,27 @@ def describe(self, timeframe, delta=0, only_distance=False):
 
         return humanized
 
+    def describe_multi(self, timeframes, only_distance=False):
+        """ Describes a delta within multiple timeframes in plain language.
+
+        :param timeframes: a list of string, quantity pairs each representing a timeframe and delta.
+        :param only_distance: return only distance eg: "2 hours 11 seconds" without "in" or "ago" keywords
+        """
+
+        humanized = ""
+        for index in range(len(timeframes)):
```

Comment:
My only comment here is that it might be beneficial to use [enumerate](https://docs.python.org/3.5/library/functions.html#enumerate) when using the index of a list in a `for` loop. However in this instance the gain is probably minimal.

Reply:
Good point, I've fixed in most recent amend because I agree it makes the code cleaner